rabbus.go

```diff
@@ -66,9 +65,11 @@ type Message struct {
 	// Key the routing key name.
 	Key string
 	// Payload the message payload.
-	Payload interface{}
+	Payload []byte
 	// DeliveryMode indicates if the is Persistent or Transient.
 	DeliveryMode uint8
+	// ContentType the message content-type.
+	ContentType string
```

Comment:
What about you create a `type ContentType string` and add functions to validate and update value?